requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
@rpenido It looks like you are running these commands with python 3.8. I tried `make requirements` on `main` and it works well on my computer. Maybe you need to run the command with python 3.10. Could you try that?

Reply:
I have the same problem with python 3.10.13.

What's your pip version?
```
pip 23.2.1 from /home/user/.pyenv/versions/3.10.13/lib/python3.10/site-packages/pip (python 3.10)
```

It only works with pip 23.0.1 (python 3.8 or 3.10)

Reply:
Ohh I see, my pip version is:

```
pip 23.1.2 from /media/xnpio/linuxDisk/xnpiochv/Documentos/edx/openedx-learning/venv/lib/python3.10/site-packages/pip (python 3.10)
```

I will try with the latest version

Reply:
If you feel this is not an issue, feel free to close this PR!

I can downgrade and work with pip 23.0.1 for this project. I just wanted to know if someone was having the same problem.

I don't think we should spend too much time on this! :smiley:

Reply:
Yep, is the version

Reply:
> If you feel this is not an issue, feel free to close this PR!
> 
> I can downgrade and work with pip 23.0.1 for this project. I just wanted to know if someone was having the same problem.
> 
> I don't think we should spend too much time on this!

Yes I agree, I got some other errors using my python 3.10. I think we can leave as is now.

Reply:
@rpenido I think they beat us to it a few days ago with https://github.com/openedx/openedx-learning/pull/83 :laughing:

Reply:
Great! Thank you for your time here @ChrisChV!
Closing!